configure.ac:  override CONFIGURE_LINE

For reproducible binaries, we need to report a constant CONFIGURE_LINE.

Upstream-Status: Pending

Signed-off-by: Joe Slater <joe.slater@windriver.com>


--- git.orig/configure.ac
+++ git/configure.ac
@@ -15,7 +15,15 @@ AC_PREREQ(2.69)
 ################################################################################
 dnl -- Process this file with autoconf to produce a configure script.
 AC_INIT
-CONFIGURE_LINE="$0 $@"
+
+dnl per reproducible-builds.org check SOURCE_DATE_EPOCH
+dnl
+if test -z "${SOURCE_DATE_EPOCH+set}" ; then
+    CONFIGURE_LINE="$0 $@"
+else
+    CONFIGURE_LINE="configure options are not available for reproducible builds"
+fi
+
 AC_CONFIG_SRCDIR([lib/device/dev-cache.h])
 AC_CONFIG_HEADERS([include/configure.h])
 
